igcollect/linux_disklatency.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""igcollect - Linux Disk I/O latency
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import sys
+from libvirt import (
+    open,
+    libvirtError,
+)
+
+def parse_args():
+
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        '--prefix', dest='prefix', type=str, default='None',
+        help='Prefix to Graphite')
+
+    parser.add_argument(
+        '--storagepool', dest='storagepool', type=str, default='None',
+        help='Storage Pool')
+
+    parser.add_argument(
+        '--storagevol', dest='storagevol', type=str, default='None',
+        help='Storage Volume')
+
+    return parser.parse_args()
+
+def main():
+    URI = 'qemu:///system'
+    args = parse_args()
+    template = args.prefix + '.{}.{} {} ' + str(int(time()))
+
+    try:
+        conn = open(URI)
+    except libvirtError as e:
+        print('Failed to open connection to qemu:///system: {}'.format(e))
+        exit(1)
+
+    try:
+        strgpool = conn.storagePoolLookupByName(args.storagepool)
+    except libvirtError as e:
+        print('Failed to find {} pool: {}'.format(args.storagepool, e))
+        exit(1)
+
+    try:
+        stgvol = strgpool.storageVolLookupByName(args.storagevol)
+    except libvirtError as e:
+        print("An exception has occured while opening the volume: {} {"
+              "}".format(args.storagevol, e))
```

Comment:
Other messages are "failed to..." but this is "an exception has occured...".

Reply:
Splitting string on {} makes it very hard readable, I was about to comment on lack of 2nd formatting parameter. The whole split generally seems to violate PEP-8.

Reply:
Also note the inconsistent use of " and ' throughout the prints. I suggest: 
```python
print(
    'An exception has occured while opening the volume: {} {}'
    .format(args.storagevol, e)
)
```

Reply:
fixed